### PR TITLE
Update terraform planner role to roles/viewer

### PR DESCRIPTION
**Pull request by KacperMalachowski**
**Description**

I wrong;y assu,e that browser role can browser data about resources. Fix it up quickly with viewer role that can view what resource state is.

Changes proposed in this pull request:

- Update role to viewer, so the planenr SA will gain access to reources not only reource list

**Related issue(s)**

/kind bug
/area automation

### CODE REVIEW

**Review by Sawthis — DISMISSED** [review]

File: configs/terraform/environments/prod/terraform-service-accounts.tf [hunk 1]
```diff
@@ -48,13 +48,33 @@ resource "google_service_account" "terraform_planner" {
...
-  role    = "roles/browser"
+  role    = each.key
   member  = "serviceAccount:${google_service_account.terraform_planner.email}"
 }
 
+resource "google_project_iam_member" "terraform_planner_prow_project_read_role" {
+  project = var.terraform_planner_gcp_service_account.project_id
+  role    = google_project_iam_custom_role.planner_read_access_role.name
+  member  = "serviceAccount:${google_service_account.terraform_planner.email}"
+}
```
Comment (dekiel) [inline configs/terraform/environments/prod/terraform-service-accounts.tf:68]: Do we still need this custom role?

**Review by dekiel — CHANGES_REQUESTED** [review]
See comment about custom role.

File: configs/terraform/environments/prod/terraform-service-accounts.tf (hunk 1 above)
Comment (KacperMalachowski) [reply configs/terraform/environments/prod/terraform-service-accounts.tf:68]: No, my fault, removed.

**Review by dekiel — APPROVED** [review]

**Pull request merged**